python/src/otel/otel_sdk/requirements-nodeps.txt

```diff
@@ -1,4 +1,6 @@
 opentelemetry-instrumentation-aiohttp-client==0.24b0
+opentelemetry-util-http==0.24b0
+asgiref~=3.0
```

Comment:
Nit, we should group these 3 dependencies together with a comment. Also we could use lexicographical where possible here (and opentelemetry-util-http==0.24b0 depends on asgiref anyways?).

```suggestion
# All these packages are required for aiohttp instrumentation

asgiref~=3.0
opentelemetry-util-http==0.24b0
opentelemetry-instrumentation-aiohttp-client==0.24b0
```